<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultadosUI</class>
 <widget class="QMainWindow" name="ResultadosUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Resultados</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="info">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="frag_label">
      <property name="text">
       <string>Estado en el nivel de fragmentación:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tablaFrag">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="boca_label">
      <property name="text">
       <string>Estado en la boca:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tablaBoca">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="KoyaguchiAnalitico_label">
      <property name="text">
       <string>Paramétros del modelo Analítico de Koyaguchi:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tablaKoyaguchiAnalitico">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="tabla_label">
      <property name="text">
       <string>Tabla:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tabla">
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionGuardar_datos"/>
    <addaction name="actionCerrar"/>
   </widget>
   <widget class="QMenu" name="menuGr_ficas">
    <property name="title">
     <string>Gráficas</string>
    </property>
    <addaction name="actionNueva"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuGr_ficas"/>
  </widget>
  <action name="actionGuardar_datos">
   <property name="text">
    <string>Guardar datos</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCerrar">
   <property name="text">
    <string>Cerrar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionNueva">
   <property name="text">
    <string>Nueva</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
